website/content/en/status/report-2023-04-2023-06/kboot.adoc

```diff
@@ -0,0 +1,48 @@
+=== LinuxBoot FreeBSD on aarch64
+
+Contact: Warner Losh <imp@bsdimp.com>
+
+Links: +
+link:https://www.linuxboot.org/[LinuxBoot Project] +
+link:https://docs.google.com/presentation/d/1N5Jp6XzYWv9Z9RhhETC-e6tFkqRHvp-ldRDW_9h2JCw/edit?usp=sharing[BSDCan 2024 kboot talk slides]
+
+LinuxBoot is an effort to create a clean, robust, auditable and repeatable boot firmware.
+What originally started as a specific project at Google has grown to encompas any boot environment that uses Linux to launch the final operating system.
```

Comment:
```suggestion
What originally started as a specific project at Google has grown to encompass any boot environment that uses Linux(R) to launch the final operating system.
```

Reply:
Not going to add (r) to Linux. It's not needed and I'm under no obligation to do so.

Reply:
Agreed. I wasn't sure, IIRC status reports are quite easygoing in this respect.